extension.js

```diff
@@ -66,8 +66,8 @@ class ForceQuitDialog extends ModalDialog.ModalDialog {
         super._init({ styleClass: 'force-quit-dialog' });
 
         const content = new Dialog.MessageDialogContent({
-            title: _('Quit applications'),
-            description: _("If an application doesn't respond for a while, select its name and click Quit Application."),
+            title: _('Force Quit an Application'),
```

Comment:
```suggestion
            title: _('Force an Application to Quit'),
```
??

Reply:
I personally like that better. "Force Quit" seems like a very jargon-y phrase.